lib/howl/variables/core_variables.moon

```diff
@@ -25,3 +25,10 @@ config.define
   description: 'Whether to automatically format code when possible'
   default: true
   type_of: 'boolean'
+
+config.define
+  name: 'preview_files'
+  description: 'Whether to automatically preview files when switching between them'
```

Comment:
The description should be a little more generic, it's not necessary that a switch will be the action taken after the interaction.